### ISSUE TITLE
[REQUEST]: Add bert-vits v2 support because its the best one for latency and stuff and if you can add gpu support

### ISSUE THREAD

**Issue by Cxsmo-ai** [deiteris/voice-changer#176]
### In a few words, describe your idea

[REQUEST]: Add bert-vits v2 support because its the best one for latency and stuff and if you can add gpu support

**Comment by deiteris** [comment]
Hi @Cxsmo-ai. Bert-VITS2 is a TTS, not STS, which I don't to intend to work with yet, and it supports GPU since day 1. It was also superseded by [fish-speech](https://github.com/fishaudio/fish-speech) which is in active development.

**Comment by Cxsmo-ai** [comment]
yes its is sts its literally Beatrice v2

**Comment by Cxsmo-ai** [comment]
yeah and fishspeech is better then why not add that but even non fork w-okada has bert-vits2

**Comment by deiteris** [comment]
Where did you read that original w-okada's version has bert-vits2? It has so-vits-svc. Though I see what you mean and it could probably be possible to use VITS2, I might as well look into fish-speech probably but I'd like to focus more on common features than introducing more code that has to be maintained.
